tools/ci-ts/test-helpers/common.ts

```diff
@@ -103,10 +103,22 @@ export async function deployContract<T extends DeployFactory>(
 
 export async function wait(ms: number) {
   return new Promise(res => {
-    setTimeout(() => res(), ms)
+    setTimeout(res, ms)
   })
 }
 
+export async function waitForService(endpoint: string, timeout = 20000) {
```

Comment:
Can we use jsdoc here? e.g what the endpoint format is, what the timeout duration resolution is